libr/core/cmd_print.c

```diff
@@ -5659,6 +5659,75 @@ l = use_blocksize;
 				}
 			}
 			break;
+		case 't': // "pst"
+			if (l > 0) {
+				ut8 *var = malloc (len);
```

Comment:
check for null. why chk first l > 0 and then use len?

Reply:
@trufae I remember I have taken another command as example before creating this function. I might be ps and it was same.